test/index.js

```diff
@@ -330,6 +349,70 @@ describe('combinator', function() {
 
   });
 
+  describe('A', function() {
+
+    it('is a binary function', function() {
+      eq(typeof S.A, 'function');
+      eq(S.A.length, 2);
+    });
+
+    it('returns the result of applying its first argument to its second', function() {
+      eq(S.A(R.add(1), 1), 2);
+      eq(R.map(function(f) { return S.A(f, 3); }, [R.add(2), R.multiply(2)]), [5, 6]);
```

Comment:
Ah right. I ended up using a lambda because I didn't want to use `flip` or `C` before they were tested. The placeholder didn't come to mind.